lxqt-base/lxqt-panel/lxqt-panel-1.0.0.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="LXQt desktop panel and plugins"
+HOMEPAGE="https://lxqt-project.org/"
+
+MY_PV="$(ver_cut 1-2)"
+
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/lxqt/${PN}.git"
+else
+	SRC_URI="https://github.com/lxqt/${PN}/releases/download/${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
+fi
+
+LICENSE="LGPL-2.1 LGPL-2.1+"
+SLOT="0"
+IUSE="+alsa colorpicker cpuload +desktopswitch +directorymenu dom +kbindicator
++mainmenu +mount networkmonitor pulseaudio +quicklaunch lm-sensors +showdesktop
++spacer +statusnotifier sysstat +taskbar tray +volume +worldclock"
+
+# Work around a missing header issue: https://bugs.gentoo.org/666278
+REQUIRED_USE="
+	|| ( desktopswitch mainmenu showdesktop taskbar )
+	volume? ( || ( alsa pulseaudio ) )
+"
+
+BDEPEND="
+	>=dev-qt/linguist-tools-5.15:5
+	>=dev-util/lxqt-build-tools-0.10.0
+	virtual/pkgconfig
+"
+DEPEND="
```

Comment:
The 'qtsvg' package was removed here, but it seems to still be required (configure fails without it present due to this line here):

https://github.com/lxqt/libqtxdg/blob/master/cmake/qt5xdgiconloader-config.cmake.in#L7

Likewise, the 'qtconcurrent' package was removed, but required still if statusnotifier is enabled:

https://github.com/lxqt/lxqt-panel/blob/master/plugin-statusnotifier/CMakeLists.txt#L5

Same for 'xcb-util' with tray enabled:

https://github.com/lxqt/lxqt-panel/blob/master/plugin-tray/CMakeLists.txt#L5

Regarding 'menu-cache', that's something I'm not sure I'd be ready to drop unless there's a clear indication somewhere that it's not being used here.  Missing it has caused runtime issues before, though I think that was with pcmanfm-qt or something like that, but for now, I'd keep it.

See also:

https://github.com/lxqt/libfm-qt/issues/417

Reply:
We should be good now!